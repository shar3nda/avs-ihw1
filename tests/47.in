8 8 2 -7 -6 -2 -8 -7 -6 -8