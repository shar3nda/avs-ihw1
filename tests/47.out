-1 7 -1 5 4 -1 5 -2 2 -2